index.css

```diff
@@ -711,6 +722,7 @@ label {
     font-weight: 400;
     text-align: justify;
     justify-content: center;
+    padding-right: 90px;
```

Comment:
:red_circle: [REQUESTED] As requested by the previous reviewer, Kindly make sure you fix the style of this paragraph to match the Figma Design. I think you can use pixels or rem for the font-size property to fix this

Your Design
![Screenshot from 2023-01-21 23-20-02](https://user-images.githubusercontent.com/26736582/213887682-f8f02654-63f6-4bc1-b27e-de8c6482f431.png)


Figma Design
![Screenshot from 2023-01-21 23-19-45](https://user-images.githubusercontent.com/26736582/213887684-9e2c7e99-9368-464e-9943-300481f21a34.png)